src/test/java/api/InstanceRelationshipsTest.java

```diff
@@ -136,6 +141,299 @@ public void canFetchMultipleInstancesWithAllRelationshipsTypes() throws Exceptio
     });
   }
 
+  @Test
+  public void cannotCreateAnInstanceWithNonExistedPrecedingTitleId()
+    throws InterruptedException, MalformedURLException, TimeoutException,
+    ExecutionException {
+
+    String precedingTitleId = UUID.randomUUID().toString();
+    final JsonObject expectedErrorResponse = new JsonObject()
+      .put("message", String.format("Cannot set preceding_succeeding_title.precedinginstanceid = " +
+        "%s because it does not exist in instance.id.", precedingTitleId));
+    precedingSucceedingTitlesClient.emulateFailure(
```

Comment:
There is record-processor approach for FakeOkapiStorage modules (e.g. support.fakes.processors.StorageRecordPreProcessors#setEffectiveCallNumberComponents), I'd use them in this case, instead of failure emulation. 

What do you think?